spec/directives/import/index/dir_dot_scss/error

```diff
@@ -0,0 +1 @@
+Is a directory @ io_fread - /sass/spec/directives/import/index/dir_dot_scss/dir.scss
```

Comment:
I think Ruby Sass should act as though it failed to find the file if it's a directory, rather than trying to read it and forwarding the IO error. That is, it should check `File.file?` rather than `File.exist?`.

Reply:
So it this work in the ruby sass repo or here?

Reply:
The behavior should be changed in the Ruby Sass repo and the expectation should be changed here.